official/resnet/keras/README.md

```diff
@@ -0,0 +1,57 @@
+This forlder contains the Keras implementation of the ResNet models. For more 
```

Comment:
Somewhere in the doc you should say CIFAR-10 will use a ResNet56 model and imagenet will use ResNet50 and link to those files.

Reply:
Done.